src/documents/activities/FunctionalMashup.md

```diff
@@ -6,39 +6,54 @@ eleventyNavigation:
     order: 8
 ---
 
-This activity can be used in a co-design or other group design session to encourage participants to work together toward more creative and resourceful solutions, by thinking beyond the current or obvious.
+This activity can be used in a co-design or other group design session to encourage participants to work together toward
+more creative and resourceful solutions, by thinking beyond the current or obvious.
 
 ## How
 
 ### Preparation
 
-You will need a deck of object cards. You can create your deck using any available materials. You might print images of different objects on paper or cardstock, use language learning flashcards, cut out magazine images and paste them onto cards, or find other ways to create a collection of random objects.
+You will need a deck of object cards. You can create your deck using any available materials. You might print images of
+different objects on paper or cardstock, use language learning flashcards, cut out magazine images and paste them onto
+cards, or find other ways to create a collection of random objects.
 <figure>
-[![An image showing a large selection of cards spread out on the floor, each with a different magazine image pasted on.](/assets/images/object_cards.jpg)](/assets/images/object_cards.jpg)
+[![An image showing a large selection of cards spread out on the floor, each with a different magazine image pasted
+on.](/assets/images/object_cards.jpg)](/assets/images/object_cards.jpg)
 <figcaption>
-*Figure 1:* Example of a collection of object cards created by pasting random magazine images onto sheets of paper.</figcaption>
+*Figure 1:* Example of a collection of object cards created by pasting random magazine images onto sheets of
+paper.</figcaption>
```

Comment:
Another case of the `<figcaption>` rendering incorrectly. I'm assuming that this is likely the same issue as above, so it would be good to review all of the `<figcaption>` elements that were touched.

It seems that a `<br>` tag is being added.

```html
<figcaption><p><em>Figure 1:</em> Example of a collection of object cards created by pasting random magazine images onto sheets of<br>
paper.</p>
</figcaption>
```

![Screen Shot 2020-12-07 at 11 50 41 AM](https://user-images.githubusercontent.com/556024/101379593-7123af00-3882-11eb-89a5-9b35e858d7ee.png)